autoload/lsp.vim

```diff
@@ -243,6 +243,8 @@ function! s:on_cursor_moved() abort
 
     if g:lsp_diagnostics_echo_cursor
         call lsp#ui#vim#diagnostics#echo#cursor_moved()
+    elseif g:lsp_diagnostics_float_cursor && lsp#ui#vim#output#float_supported()
```

Comment:
@tyru this shouldn't be else if since if the user has both `g:lsp_diagnostics_echo_cursor` and `g:lsp_diagnostics_float_cursor` enabled only echo cursor works currently.

Reply:
ah I thought the variables are exclusive but it is unnecessary that they are exclusive.
I will fix them.

Reply:
@prabirshrestha sent #711